resolwe_bio/processes/import_data/metadata.py

```diff
@@ -148,6 +173,189 @@ def run(self, inputs, outputs):
         if n_samples < 1:
             self.error("The uploaded table contains no samples.")
 
+        df_data = convert_to_data_frame(str(path))
+
+        sample_columns = {"mS#Sample ID", "mS#Sample name", "mS#Sample slug"}
+        if not sample_columns.intersection(set(df_data.columns)):
+            self.error(
+                f"Missing sample column. The uploaded Orange table should "
+                f"contain one of the following columns: {sorted(sample_columns)}."
+            )
+
+        samples = Entity.filter(collection_id=collections[0].id)
+
+        if "mS#Sample ID" in df_data.columns:
+            col_samples = set([sample.id for sample in samples])
+            sample_header = "mS#Sample ID"
+            intersection = col_samples.intersection(set(df_data[sample_header].values))
```

Comment:
No need to repeat this line 3 times. Just write once after this if-elif clause is done.